plutus-tx-plugin/test/Plugin/Primitives/Spec.hs

```diff
@@ -45,21 +45,21 @@ primitives = testNested "Primitives" [
   , goldenPir "ifThenElse" ifThenElse
   , goldenUEval "ifThenElseApply" [ toUPlc ifThenElse, toUPlc int, toUPlc int2 ]
   , goldenPir "emptyByteString" emptyByteString
-  , goldenUEval "emptyByteStringApply" [ getPlc emptyByteString, liftProgram Builtins.emptyByteString ]
+  , goldenUEval "emptyByteStringApply" [ getPlcNoAnn emptyByteString, liftProgram Builtins.emptyByteString ]
```

Comment:
But the `lift*` functions can't know the source locations, so it would always be `mempty`, right?

Reply:
Right, or `Default` or we could pass it as an argument... The only real advantage would be that you'd get something out with a default-annotation-of-your-type, rather than having to convert your existing AST to have no annotations. Really not a big deal and unclear if it would even be nicer.

Reply:
Hmm let's punt on this. I'm still not sure whether every annotation type should necessarily have a `Default` instance.